charts/reform-scan-notification-service/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for Reform Scan Notification Service
 name: reform-scan-notification-service
 home: https://github.com/hmcts/reform-scan-notification-service
-version: 0.0.1
+version: 0.0.2
```

Comment:
is it fluxed already? i missed the prs for this one. if not then no need to bump the version

Reply:
yup flux has gone in now with 0.0.2

Reply:
ah cool. then i just missed it ;)